bundles/org.openhab.binding.mideaac/src/main/java/org/openhab/binding/mideaac/internal/cloud/Cloud.java

```diff
@@ -203,7 +206,7 @@ public Cloud(String email, String password, CloudProvider cloudProvider, HttpCli
             if (code != 0) {
                 errMsg = msg;
                 logger.warn("Error {} logging to Cloud: {}", code, msg);
```

Comment:
Now this logger can be changed to debug as the exception is thrown and handled upstream.